app/helpers/shows_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ShowsHelper
+end
```

Comment:
Optional: Good practice to not add boilerplate code like this unless it's actually being used